src/main/java/org/broadinstitute/hellbender/utils/read/FlowBasedReadUtils.java

```diff
@@ -26,17 +36,35 @@ public class FlowBasedReadUtils {
     static public class ReadGroupInfo {
         final public String  flowOrder;
         final public int     maxClass;
-
+        final public boolean isFlowPlatform;
         private String  reversedFlowOrder = null;
 
         public ReadGroupInfo(final SAMReadGroupRecord readGroup) {
+            if (readGroup.getPlatform()==null){
+                isFlowPlatform = false;
+            } else if (NGSPlatform.fromReadGroupPL(readGroup.getPlatform())==NGSPlatform.UNKNOWN){
+                isFlowPlatform = false;
+            } else if (NGSPlatform.fromReadGroupPL(readGroup.getPlatform()) == NGSPlatform.LS454) {
```

Comment:
I know 454 data is ancient at this point but i would still prefer this LS454 case to explicitly check for 
```
if (readGroup.getFlowOrder()!=null) {
                    isFlowPlatform = true; else false;
```
I'm not sure i trust the Ultima flow ordering code is going to handle 454 data well without testing it

Reply:
I think this is not this check is supposed to do. Old Ultima data were labeled LS454 and initially did not have the FO tag. Afterwards the tag was added .
The full check is done by isFlowPlatform function that also verifies that the reads have tp tag.

Reply:
Ahh i see... horrifying... well i'm alright to let this through in this case then